src/panels/lovelace/components/hui-image.ts

```diff
@@ -161,6 +161,15 @@ export class HuiImage extends LitElement {
       imageSrc = this.image;
     }
 
+    if (
+      this._loadState === LoadState.Loading &&
+      !this.cameraImage &&
+      imageSrc &&
+      !imageSrc.startsWith("http")
+    ) {
```

Comment:
I think it might be enough to show the spinner just for camera images

Reply:
I fully agree on this! However I wanted to provide minimum impact with my PR...

Do you think I can just do:
```
      this._loadState === LoadState.Loading &&
      !this.cameraImage &&
      imageSrc
    ) {
      this._loadState = LoadState.Loaded;
```
?

(I would read this as "if the image is still loading but it's not from camera and there is an actual image URL, then consider the image as already loaded")

Reply:
We should move it to the rendering then

Reply:
I pushed this change. 

If you mean to change the rendering part (strictly speaking, ie. where the html is returned), I think there are more lines of code and checks to change there, rather than just change the state to "loaded" as I did so I'm not very confident to perform this change myself (also, I have no cameras, so it will be hard for me to check).

Reply:
Add `this.cameraImage` on line 244: <https://github.com/home-assistant/frontend/pull/10086/files#diff-9e43694b64739d4f3f9ae00b65811d9954b15a6756aa734512c2877d9e909a04R244>

Reply:
I think there are multiple places where the concept of "Loading" vs "Loaded" is used, eg.:
```
          filter:
            this._loadState === LoadState.Loaded || this.cameraView === "live"
              ? filter
              : undefined,
```
```
                  display:
                    useRatio || this._loadState === LoadState.Loaded
                      ? "block"
                      : "none",
```

For this reason I preferred to maintain the logic (that is proved to be working) of having "Loaded" images managed as they are and force the state from "Loading" to "Loaded" for images that we know should never be in "Loading" state (ie. everything but cameras).

Reply:
I see your point 👍